openshift/e2e-tests-openshift.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/sh
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
+
+set -x
+
+readonly API_SERVER=$(oc config view --minify | grep server | awk -F'//' '{print $2}' | awk -F':' '{print $1}')
+readonly OPENSHIFT_REGISTRY="${OPENSHIFT_REGISTRY:-"registry.svc.ci.openshift.org"}"
+readonly TEST_NAMESPACE=build-pipeline-tests
+readonly TEST_YAML_NAMESPACE=build-pipeline-tests-yaml
+readonly BUILD_PIPELINE_NAMESPACE=knative-build-pipeline
+readonly IGNORES="git-volume|gcs-archive|docker-basic"
+
+env
+
+function install_build_pipeline(){
+  header "Installing Knative Build Pipeline"
+  # Grant the necessary privileges to the service accounts Knative will use:
+  oc adm policy add-scc-to-user anyuid -z build-pipeline-controller -n knative-build-pipeline
+  oc adm policy add-cluster-role-to-user cluster-admin -z build-pipeline-controller -n knative-build-pipeline
+
+  create_build_pipeline
+
+  wait_until_pods_running $BUILD_PIPELINE_NAMESPACE || return 1
+
+  header "Knative Build Pipeline Installed successfully"
+}
+
+function create_build_pipeline(){
+  resolve_resources config/ build-pipeline-resolved.yaml
+  oc apply -f build-pipeline-resolved.yaml
+}
+
+function resolve_resources(){
```

Comment:
@mvinkler in his PR for this branch (but OCP 3.11) had this function different: https://github.com/openshift/knative-build-pipeline/pull/4/commits/33805aa0309f332b8d76212034197b32f611b700
I think there's something missing.